### ISSUE TITLE
bitcoin.update.sh update

### ISSUE THREAD

**Issue by rootzoll** [raspiblitz/raspiblitz#3852]
the bitcoin update needs multiple refactoring:

- [x] it needs to run non-interactive for provisioning
- [x] it should just have a list if trusted signers and check if if those signed the release
- [x] the "tested" should also work if the numer if stored in conf ins LOWER then the tested

last point is so that there could be multiple updates if tested version before a new sd card

**Comment by rootzoll** [comment]
Test with offering bitcoin v25.0 as update option [edit: OLD]

**Comment by rootzoll** [comment]
Also got a error report when somebody updating to bitcoin 25

```
/dev/fd/63: line 22: bitcoin-25.0-aarch64-linux-gnu.tar.gz:: command not found
/dev/fd/63: line 29: bitcoin-25.0/: No such file or directory
/dev/fd/63: line 30: bitcoin-25.0/[README.md](https://readme.md/): No such file or directory
/dev/fd/63: line 31: bitcoin-25.0/bin/: No such file or directory
/dev/fd/63: line 32: bitcoin-25.0/bin/bitcoin-cli: No such file or directory
/dev/fd/63: line 33: bitcoin-25.0/bin/bitcoin-qt: No such file or directory
/dev/fd/63: line 34: bitcoin-25.0/bin/bitcoin-tx: No such file or directory
/dev/fd/63: line 35: bitcoin-25.0/bin/bitcoin-util: No such file or directory
/dev/fd/63: line 36: bitcoin-25.0/bin/bitcoin-wallet: No such file or directory
/dev/fd/63: line 37: bitcoin-25.0/bin/bitcoind: No such file or directory
/dev/fd/63: line 38: bitcoin-25.0/bin/test_bitcoin: No such file or directory
/dev/fd/63: line 39: bitcoin-25.0/bitcoin.conf: No such file or directory
/dev/fd/63: line 40: bitcoin-25.0/include/: No such file or directory
/dev/fd/63: line 41: bitcoin-25.0/include/bitcoinconsensus.h: No such file or directory/dev/fd/63: line 42: bitcoin-25.0/lib/: No such file or directory
/dev/fd/63: line 43: bitcoin-25.0/lib/[libbitcoinconsensus.so](https://libbitcoinconsensus.so/): No such file or directory/dev/fd/63: line 44: bitcoin-25.0/lib/[libbitcoinconsensus.so](https://libbitcoinconsensus.so/).0: No such file or directory
/dev/fd/63: line 45: bitcoin-25.0/lib/[libbitcoinconsensus.so](https://libbitcoinconsensus.so/).0.0.0: No such file or directory
/dev/fd/63: line 46: bitcoin-25.0/share/: No such file or directory
/dev/fd/63: line 47: bitcoin-25.0/share/man/: No such file or directory
/dev/fd/63: line 48: bitcoin-25.0/share/man/man1/: No such file or directory
/dev/fd/63: line 49: bitcoin-25.0/share/man/man1/bitcoin-cli.1: No such file or directory
/dev/fd/63: line 50: bitcoin-25.0/share/man/man1/bitcoin-qt.1: No such file or directory
/dev/fd/63: line 51: bitcoin-25.0/share/man/man1/bitcoin-tx.1: No such file or directory
/dev/fd/63: line 52: bitcoin-25.0/share/man/man1/bitcoin-util.1: No such file or directory
/dev/fd/63: line 53: bitcoin-25.0/share/man/man1/bitcoin-wallet.1: No such file or directory
/dev/fd/63: line 54: bitcoin-25.0/share/man/man1/bitcoind.1: No such file or directory
/dev/fd/63: line 55: bitcoin-25.0/share/rpcauth/: No such file or directory
/dev/fd/63: line 56: bitcoin-25.0/share/rpcauth/[README.md](https://readme.md/): No such file or directory
/dev/fd/63: line 57: bitcoin-25.0/share/rpcauth/[rpcauth.py](https://rpcauth.py/): No such file or directory
/dev/fd/63: line 58: Bitcoin: command not found
Green activity light stays dark and LCD turns white when shutdown complete.
It will then reboot again automatically.
-----------------------------------------------
stop electrs - please wait ..
stop LNDg - please wait ..
```

**Comment by rootzoll** [comment]
Rewrite for a v1.10.1 or v1.11.0 release

**Comment by rootzoll** [comment]
I think this was solved in v1.11.1 ... but recheck

**Comment by rootzoll** [comment]
OK tested with v1.11.1 - already all good. Closing issue.
